Jenkinsfile

```diff
@@ -281,6 +281,16 @@ try {
         }
       }
     },
+    'GPU: Jetson - ARM': {
```

Comment:
Rename to armv8

Reply:
Please do not comment until WIP removed.